mercure.rst

```diff
@@ -568,6 +568,35 @@ sent. Here is the PublisherStub implementation::
     App\Tests\Functional\Fixtures\PublisherStub:
         decorates: mercure.hub.default.publisher
 
+Debugging
+---------
+
+The MercureBundle now integrates a WebProfiler panel. To enable it, update your configuration as following:
```

Comment:
I think we can remove the "now", as the documentation is a guide and not so much of a changelog. We can add a `.. versionadded:: 0.2` "The WebProfiler panel was introduced in MercureBundle 0.2" directive at the start of this chapter though

Reply:
`.. versionAdded:: 0.2` doesn't seem to be accepted by the linter:
>mercure.rst ✘
>  571: You are not allowed to use version "0.2". Only major version "5" is allowed.
>   ->  .. versionadded:: 0.2
>  571: Please only provide ".. versionadded::" if the version is greater/equal "5.0"
>   ->  .. versionadded:: 0.2

Reply:
Thanks for your very quick responses! 😍 You can ignore this lint message

@OskarStark what would you suggest doing here?

Reply:
I will add it to the ignore while merging and fix the `versionadded` directive 👍